### PR TITLE
added package

### PR DESCRIPTION
**Pull request by mounikanimmu**

### CODE REVIEW

**Review by davidgladson — APPROVED** [review]

File: src/app/routes/email-verified/email-verified.component.html [hunk 1]
```diff
@@ -39,8 +39,14 @@ <h1>Follow us on:</h1>
         </div>
         <div class="col-12 col-md-6 py-2">
           <ul class="nav justify-content-end">
+            <div class="d-md-block d-none containera px-lg-5">
+              <button mat-button onclick="location.href='privacy'" class="foo" >Privacy Policy</button>&nbsp;
```
Comment (phoenixnitin) [inline src/app/routes/email-verified/email-verified.component.html:43]: what is class foo

File: src/app/routes/email-verified/email-verified.component.scss [hunk 2]
```diff
@@ -38,6 +39,16 @@
    button{
      background-color: #ED3237;
      color:white;
-     left:18%;
+     left:12%;
+   }
+   .foo{
```
Comment (phoenixnitin) [inline src/app/routes/email-verified/email-verified.component.scss:44]: provide better name

File: src/app/routes/resetpwd/resetpwd.component.html [hunk 3]
```diff
@@ -1,12 +1,12 @@
 <!--Section: Block Content-->
-<body marginheight="0" topmargin="0" marginwidth="0" leftmargin="0">
+
+<div class="d-md-block d-none containera px-lg-3">
     <!--100% body table-->
-    <table cellspacing="0" border="0" cellpadding="0" width="100%" bgcolor="#f2f3f8"
-        style="@import url(https://fonts.googleapis.com/css?family=Rubik:300,400,500,700|Open+Sans:300,400,600,700); font-family: 'Open Sans', sans-serif;">
+    <table
+              style="@import url(https://fonts.googleapis.com/css?family=Rubik:300,400,500,700|Open+Sans:300,400,600,700); font-family: 'Open Sans', sans-serif;">
```
Comment (phoenixnitin) [inline src/app/routes/resetpwd/resetpwd.component.html:6]: import into styles or index file, create a class to apply font

File: src/app/routes/resetpwd/resetpwd.component.html [hunk 4]
```diff
@@ -97,8 +97,106 @@ <h1>You have requested to reset your password</h1>
             </td>
         </tr>
     </table>
+    </div>
     <!--/100% body table-->
+    <div class="d-block d-md-none rwdcontainer reset">
+      <table
+      style="@import url(https://fonts.googleapis.com/css?family=Rubik:300,400,500,700|Open+Sans:300,400,600,700); font-family: 'Open Sans', sans-serif;">
```
Comment (phoenixnitin) [inline src/app/routes/resetpwd/resetpwd.component.html:104]: again, create class to apply this, importing from url should be done in styles or index file

File: src/app/routes/resetpwd/resetpwd.component.html [hunk 5]
```diff
@@ -97,8 +97,106 @@ <h1>You have requested to reset your password</h1>
...
+                </td>
+            </tr>
+            <tr>
+                <td style="height:20px;">&nbsp;</td>
+            </tr>
+            <tr>
+                <td>
+                    <table class="table">
+                        <tr>
+                            <td style="height:40px;">&nbsp;</td>
```
Comment (phoenixnitin) [inline src/app/routes/resetpwd/resetpwd.component.html:124]: do not do nbsp, remove from everywhere and use padding, margin, min-width or min-height

File: src/app/routes/resetpwd/resetpwd.component.html [hunk 6]
```diff
@@ -97,8 +97,106 @@ <h1>You have requested to reset your password</h1>
...
+                        <div class="example-button-container" style="float:none;">
+                            <button style="left:10%;background-color: #ED3237;" mat-fab color="warn" aria-label="Example icon button with a home icon">
+                              <mat-icon><svg style="margin-top:-16px;" xmlns="http://www.w3.org/2000/svg" width="26" height="26" fill="currentColor" class="bi bi-hand-thumbs-up-fill" viewBox="0 0 16 16">
+                                <path d="M6.956 1.745C7.021.81 7.908.087 8.864.325l.261.066c.463.116.874.456 1.012.965.22.816.533 2.511.062 4.51a9.84 9.84 0 0 1 .443-.051c.713-.065 1.669-.072 2.516.21.518.173.994.681 1.2 1.273.184.532.16 1.162-.234 1.733.058.119.103.242.138.363.077.27.113.567.113.856 0 .289-.036.586-.113.856-.039.135-.09.273-.16.404.169.387.107.819-.003 1.148a3.163 3.163 0 0 1-.488.901c.054.152.076.312.076.465 0 .305-.089.625-.253.912C13.1 15.522 12.437 16 11.5 16H8c-.605 0-1.07-.081-1.466-.218a4.82 4.82 0 0 1-.97-.484l-.048-.03c-.504-.307-.999-.609-2.068-.722C2.682 14.464 2 13.846 2 13V9c0-.85.685-1.432 1.357-1.615.849-.232 1.574-.787 2.132-1.41.56-.627.914-1.28 1.039-1.639.199-.575.356-1.539.428-2.59z"/>
```
Comment (phoenixnitin) [inline src/app/routes/resetpwd/resetpwd.component.html:175]: is it really required to create icon like this

File: src/app/routes/resetpwd/resetpwd.component.html [hunk 7]
```diff
@@ -97,8 +97,106 @@ <h1>You have requested to reset your password</h1>
...
+                              </svg></mat-icon>
+                            </button>
+                          </div>
+                       </tr>
+                        <tr>
+                            <td style="height:40px;">&nbsp; </td>
```
Comment (phoenixnitin) [inline src/app/routes/resetpwd/resetpwd.component.html:181]: remove nbsp

**Review by phoenixnitin — CHANGES_REQUESTED** [review]
Please update code according to comments

**Review by phoenixnitin — APPROVED** [review]
Please work on changes, approving for now

**Pull request merged**